tsconfig.production.json

```diff
@@ -4,7 +4,7 @@
     "module": "commonjs",
     "target": "es2015",
     "declaration": true,
-    "outDir": "./dist",
+    "outDir": "./build",
```

Comment:
No need to change name of result dir.
This folder will be published as npm package.
For example, try opening `node_modules` and see what it looks like there.

Also rename this file to `tsconfig.json` and after that fix build command